android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -810,12 +857,14 @@ private MediaSource buildTextSource(String title, Uri uri, String mimeType, Stri
 
     private void releasePlayer() {
         if (player != null) {
+            adsLoader.setPlayer(null);
             updateResumePosition();
             player.release();
             player.removeListener(this);
             trackSelector = null;
             player = null;
         }
+        adsLoader.release();
```

Comment:
would be better to adsLoader to null to allow garbage collector to clear memory

Reply:
I'm not sure about this one, what do you want me to do?
```suggestion
        adsLoader = null;
```

or

```suggestion
        adsLoader.setPlayer(null);
```
like in the condition above
or

```suggestion
        adsLoader.release();
        adsLoader = null;
```
?

Reply:
I think the 3rd one would be better !

Reply:
Alright, I made this change! 😉

Reply:
As this is my first PR I'm not sure how to deal with the conversations, should I resolve them myself or you will once you think the corrections are ok?

Reply:
You put them as closed, I will do another review round & some tests in sample app before my final approval :)